### PR TITLE
Add a job for Ubuntu trusty

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,13 @@ env:
     - ORACLE_HOME=/u01/app/oracle/product/11.2.0/xe
     - ORACLE_SID=XE
 
+matrix:
+  include:
+    - dist: precise
+      env: PRECISE=1
+    - dist: trusty
+      env: TRUSTY=1
```
Comment (cbandy) [inline .travis.yml]: I'd prefer `DIST=precise` and `DIST=trusty`. Any objections?

**Comment by cbandy** [comment]
Fixed in e6aeff2c5f2c54d335b0ea0c0be0add8adbda5e2.

**Comment by jayvdb** [comment]
Note I'd very much appreciate it if you re-add me as the author of e6aeff2..  I need to report my OSS work to my boss, and it is a lot easier if my commits have my name on them.

**Comment by cbandy** [comment]
Because I closed #7 first, the patch here would no longer apply.

> Note I'd very much appreciate it if you re-add me as the author of e6aeff2..  I need to report my OSS work to my boss, and it is a lot easier if my commits have my name on them.

I won't amend this commit as it is already on the master branch. Please feel free to have your boss reach out to me at bandy.chris@gmail.com, and I can explain the situation.

**Comment by jayvdb** [comment]
I understand why you amended my patch, in order to rebase it, but you also removed me as the author in the process.  In github, you can push amendments back into the PR before merging them, thereby keeping the authorship.
Appreciate you not wanting to rewrite history; this patch isnt super important anyway.  And from a legal perspective, it is not necessary to acknowledge authorship with the license you've chosen.   No worries here.  The next one will be the important one, and a new version should be coming quite soon.  If it needs amendment, let me know and I'll amend it. ;-)

**Pull request closed without merging**